infrastructure/foreman-configuration/foreman-server-instance-user-data.tpl.sh

```diff
@@ -28,6 +28,8 @@ docker run \\
        -e DATABASE_NAME=${database_name} \\
        -e DATABASE_USER=${database_user} \\
        -e DATABASE_PASSWORD=${database_password} \\
+       -e ELASTICSEARCH_HOST=${elasticsearch_host} \\
+       -e ELASTICSEARCH_PORT=80 \\
```

Comment:
Why hardcoded?

Reply:
Because it's not available as a parameter from terraform, but I just changed that.